spec/registry.md

```diff
@@ -255,3 +255,502 @@ which only expects the `accord` option:
 > .input {$object :noun case=nominative}
 > {{You see {$color :adjective article=indefinite accord=$object} {$object}!}}
 >```
+
+# Default Registry for LDML45
+
+This section describes the functions which each implementation MUST provide to be conformant with
+the LDML45 Technical Preview of MessageFormat v2.
+
+## String Value Selection and Formatting
+
+(Pending)
+
+## Numeric Value Selection and Formatting
+
+### Functions
+
+The following functions use numeric selection:
+
+The function `:number` is the default selector for numeric values.
+
+The function `:integer` provides a reduced set of options for selecting
+and formatting numeric values as integers.
+
+### Operands
+
+The _operand_ of a number function is either an implementation-defined type or
+a literal that matches the `number-literal` production in the [ABNF](/main/spec/message.abnf).
+All other values produce a _Selection Error_ when evaluated for selection
+or a _Formatting Error_ when attempting to format the value.
```

Comment:
It's probably fine to not mention that these values will first emit a _Resolution Error_, and then because the whole expression resolves to a fallback value, we emit the above-mentioned further errors. Or should some of that be mentioned here?

Reply:
No, we should just say _Resolution Error_ here. Implementers need to know what to throw/emit and that's what they'll emit when parsing the literal.

```suggestion
All other values produce a _Resolution Error_.
```